Resources/Prototypes/Loadouts/Jobs/Medical/seniorPhysician.yml

```
@@ -34,6 +34,33 @@
   items:
     - ClothingBeltMedicalAdvancedFilled
 
+- type: loadout
+  id: LoadoutSeniorPhysicianBluespaceBeaker
+  category: JobsMedicalSeniorPhysician
+  cost: 0
+  exclusive: true
+  requirements: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)